aws/resource_aws_spot_fleet_request.go

```diff
@@ -191,6 +191,11 @@ func resourceAwsSpotFleetRequest() *schema.Resource {
 							ForceNew: true,
 							Optional: true,
 						},
+						"iam_instance_profile_arn": {
```

Comment:
Can you please
* Document the new argument in the argument reference near the existing `iam_instance_profile` argument
* Add a simple acceptance test to cover this new argument? Copy pasting an existing test and test configuration that uses `iam_instance_profile` is fine 👍

Reply:
Documented the new argument under `launch_specification` section and added the acceptance test.